src/sources/aws_s3/sqs.rs

```diff
@@ -88,10 +97,13 @@ pub(super) struct Config {
     ///
     /// Defaults to the number of available CPUs on the system.
     ///
-    /// Should not typically need to be changed, but it can sometimes be beneficial to raise this value when there is a
-    /// high rate of messages being pushed into the queue and the objects being fetched are small. In these cases,
-    /// System resources may not be fully utilized without fetching more messages per second, as the SQS message
-    /// consumption rate affects the S3 object retrieval rate.
+    /// Should not typically need to be changed, but it can sometimes be beneficial to raise this
+    /// value when there is a high rate of messages being pushed into the queue and the objects
+    /// being fetched are small. In these cases, system resources may not be fully utilized without
+    /// fetching more messages per second, as the SQS message consumption rate affects the S3 object
+    /// retrieval rate.
+    #[configurable(metadata(docs::type_unit = "tasks"))]
```

Comment:
Thoughts on `tasks` as the unit here, I couldn't think of anything better. Maybe could reuse `connections`?

Reply:
I'm ok with tasks. I am not aware of any reason to avoid creating new ones. I think re using connections is less cohesive to what it does based on the doc comemnt.